ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -340,13 +340,14 @@ def _basic_min_x_max(draw):
             available_dtypes=helpers.get_dtypes("numeric"),
         )
     )
-    min_val = draw(helpers.array_values(dtype=dtype[0], shape=()))
-    max_val = draw(
-        helpers.array_values(dtype=dtype[0], shape=()).filter(lambda x: x > min_val)
-    )
+    if random.choice([True, False]):
+        min_val = draw(helpers.array_values(dtype=dtype[0], shape=()))
+        max_val = None
+    else:
+        min_val = None
+        max_val = draw(helpers.array_values(dtype=dtype[0], shape=()))
```

Comment:
There's no need to use the `random` module for this. It's possible to generate `None` value using hypothesis in the following way:

```python
min_val = draw(st.one_of(st.just(None), helpers.array_values(dtype=dtype[0], shape=()))
max_val = draw(st.one_of(st.just(None), helpers.array_values(dtype=dtype[0], shape=()).filter(lambda x: x > min_val)))
```